hexrd/unitcell.py

```diff
@@ -747,11 +747,20 @@ def InitializeInterpTable(self):
 
                 Z = self.atom_type[i]
                 elem = constants.ptableinverse[Z]
-                gid = fid.get('/'+elem)
-                data = np.array(gid.get('data'))
-                self.pe_cs[elem] = interp1d(data[:, 7], data[:, 3])
-                data = data[:, [7, 1, 2]]
-                f_anomalous_data.append(data)
+
+                wav = np.linspace(1.16E2, 2.86399992e-03, 189)
+                zs = np.ones_like(wav)*Z
+                zrs = np.zeros_like(wav)
+                data_zs = np.vstack((wav, zs, zrs)).T
```

Comment:
If this section of code is only used by the `else` block, maybe we should move it into the `else` block?

Reply:
done

Reply:
Looks like it isn't pushed yet.